src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,6 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
+public class ColorSupplier extends RandoM {
```

Comment:
The class `ColorSupplier` should have a method that supplies a color. Currently, it's empty and does not fulfill its purpose. Consider implementing a method that returns a random color.